app/js/utils/webAPI.js

```diff
@@ -3,9 +3,11 @@ import Auth from '../auth';
 import fetch from 'isomorphic-fetch';
 import isEmpty from './isEmpty';
 
+import $ from 'jquery';
+
 const requestPath = (path, method, data = {}) => {
   if (method === 'GET' && !isEmpty(data)) {
-    return path + '?' + encodeURIComponent(JSON.stringify(data));
+    return path + '?' + $.param(data);
```

Comment:
We're pushing back on the use of `jquery` so as to work towards removing it completely. It's a really heavy library if its purpose in this file would be limited to encoding param